lib/optimist-config-file.js

```diff
@@ -106,7 +106,42 @@ optimist.parse = function(args) {
 
 optimist.setConfigFile = function(configFileParam) {
 	this.configFileParam = configFileParam;
-	return this.describe(this.configFileParam, 'uses JSON or YAML-formatted config file to set parameters').string(this.configFileParam);
+	return this;
+};
+
+var _describe = optimist.describe,
+	_help = optimist.help,
+	headers = [],
+	nextHeader;
+
+optimist.header = function(s) {
+	nextHeader = s;
+	return this;
+};
+optimist.describe = function(opt) {
+	if (nextHeader) {
+		headers.push([nextHeader, opt]);
+		nextHeader = null;
+	}
+	return _describe.apply(optimist, arguments);
+};
+optimist.help = function() {
+	var text = _help.apply(optimist, arguments),
+		i, s, p, opt_header = "\nOptions:",
+		opt_header_len = opt_header.length;
+	//console.log(text,headers);
+	for (i = 0; i < headers.length; i++) {
+		s = "\n  --" + headers[i][1];
+		p = text.indexOf(s);
+		if (p != -1) {
+			s = "\n" + headers[i][0] + ":";
```

Comment:
What if we emit an empty line before the options group name and make it "bold" by calling `bright` from `ansistyles` module?